projects/m1/030-maximum-integer/js/index.js

```diff
@@ -0,0 +1,34 @@
+function generateRandomNumber( integer ){
+    return Math.floor( Math.random() * integer ) + 1;
+}
+
+function findMaxNumberAndCountUpdates( numbersCount ){
+    let maxNumber = 0;
+    let updates = 0;
+
+    for (let i = 0; i < numbersCount; i++) {
+
+        const currentNumber = generateRandomNumber( numbersCount );
+
+        if ( maxNumber < currentNumber || i === 0 ){
+            maxNumber = currentNumber ;
+            console.log( currentNumber + ' <== Update' );
+            updates++;
+        } else {
+            console.log( currentNumber );
+        }
+    }
+
+    return [ maxNumber, updates ];
+}
+
+function init(){
+    const numbersCount = 100;
+    const [ maxNumber, updates ] = findMaxNumberAndCountUpdates( numbersCount );

```

Comment:
Great job using **destructuring assignment**! :-)

Reply:
Thank you,
can you kindly resolve the pr?